hypervisor/arch/x86/cpu.c

```diff
@@ -447,7 +447,9 @@ void cpu_dead(void)
 		/* clean up native stuff */
 		vmx_off();
 
+		clac();
 		flush_cache_range((void *)get_hv_image_base(), CONFIG_HV_RAM_SIZE);
```

Comment:
This call sequence of clac and stac is wrong.

Reply:
Fixed